cv-data-service-library/src/main/java/com/trihydro/library/helpers/TimGenerationHelper.java

```diff
@@ -739,13 +739,13 @@ private DataFrame getDataFrame(TimUpdateModel aTim, Milepost anchor, boolean res
 
         // DataFrame
         DataFrame df = new DataFrame();
-        df.setNotUsed(aTim.getNotUsed());
+        df.setDoNotUse1(aTim.getDoNotUse1());
```

Comment:
question: with these now being set to "doNotUsexxx", can we drop them?

Reply:
Yep, these can be safely removed